setup.cfg

```diff
@@ -15,28 +15,31 @@ classifiers =
 packages = find:
 python_requires = >=3.8
 install_requires =
-    certifi >= 14.5.14
-    frozendict ~= 2.3.4
-    setuptools >= 21.0.0
-    typing-extensions ~= 4.3.0
-    urllib3 ~= 1.26.7
-    python-dateutil
-    absl-py
-    tqdm
-    jinja2
-    numpy
-    scipy
-    matplotlib
-    xarray
-    vtk
-    imageio
-    pyvista
-    imageio-ffmpeg
-    ipython
-    nglview
-    MDAnalysis
-    utm
-    dask
+  certifi >= 14.5.14
+  frozendict ~= 2.3.4
+  setuptools >= 21.0.0
+  typing-extensions ~= 4.3.0
+  urllib3 ~= 1.26.7
+  python-dateutil
+  absl-py
+  tqdm
+  jinja2
+  numpy
+  scipy
+  matplotlib
+  xarray
+  vtk
+  imageio
+  pyvista
+  imageio-ffmpeg
+  ipython
+  utm
+  dask
+
+[options.extras_require]
+molecules =
```

Comment:
Maybe call it molecules_viz. Your idea of splitting the visualization extra requirements separate by subfield (molecules, fluids) might make sense, such that users from different fields don't need to install visualization libraries only used for other fields... let's see if it works well / it's not too much fragmentation. And then in any case we might move more things to the server side.

Reply:
I think `molecules` is good enough, since it installs related with the molecules vertical and avoids long names. On the documentation, we can specify that `pip install inductiva` just installs the barebones for running simulations.

Furthermore, I am not completely sure about the fragmentation as well. There are some packages that are just specific for the FluidTank scenario for example. So the naming for fluids may be more treaky.

Reply:
FluidTank is using xarrays to deal with the output data of another project we had. At the time it was merged into the API and we were still tinkering with having frameworks to read the data into appropriate structures. Thereafter, xarray are used in combination with matplotlib to render things.

In the other, SPH scenarios we are using pyvista to hold data and render things.

Reply:
OK! We need to think deeper about this. @david451lima : you will also be helping in this front, so have a look at this dependency too later.